app/js/lib/foxbox/services/light.js

```diff
@@ -15,11 +15,13 @@ export default class LightService extends BaseService {
   }
 
   isAvailable() {
-    return this.get('available').then((response) => response.OnOff === 'On');
+    return this.fetch('device/available')
+      .then((response) => response.OnOff === 'On');
```

Comment:
With new ```supports_fetch``` signature we can eventually simplify this even more to smth like ```.then((response) => response === 'On');```, but let's wait for the foxbox to stabilize this time.